### ISSUE TITLE
Subscribe to an array events

### ISSUE THREAD

**Issue by travikk** [esp/esp-js#162]
Pass in an array of events you're interested in, or get a fat pipe of all events

**Comment by raybooysen** [comment]
You can get the fat pipe on router bye using:

```
    observeEventsOn<TModel>(modelId: string, model : TModel, methodPrefix?: string) : Disposable;
```

**Comment by travikk** [comment]
Sorry, wrote this issue without actual details.

The need here is to get an Observable that will observe an array of events, or a fat pipe. 

observeEventsOn doesn't give you an Observable but a subscription (I assume) instead, therefore provides no way of further processing the pipe in a nice way.

**Comment by KeithWoods** [comment]
Fixed via #171  and  #178 , release 2.0
